Sources/SwiftInspectorCore/Tests/TypealiasAnalyzerSpec.swift

```diff
@@ -105,6 +105,36 @@ final class TypealiasAnalyzerSpec: QuickSpec {
         }
       }
 
+      context("with multiple typealias with the same name") {
+        beforeEach {
+          fileURL = try! Temporary.makeFile(
+            content: """
+                     typealias Foo = Bar
+
+                     struct MyNamespace {
+
+                       typealias Foo = Bar
+
+                     }
+                     """
+          )
+        }
+        
+        it("returns all the names of the typeliases") {
+          let result = try? sut.analyze(fileURL: fileURL)
+          expect(result?.count) == 2
+          expect(result?.first?.name) == "Foo"
+          expect(result?.last?.name) == "Foo"
+        }
+
+        it("returns all the typealias identifiers") {
+          let result = try? sut.analyze(fileURL: fileURL)
+          expect(result?.count) == 2
+          expect(result?.first?.identifiers) == ["Bar"]
+          expect(result?.last?.identifiers) == ["Bar"]
+        }
+      }
+
```

Comment:
My philosophy on these type of cases is that the Analyzer should return everything it "sees" and then the Command should be in charge of filtering the output to be consumed by the user. That's why in here we return twice the same information, but in the Command we actually filter to display it to the user once since duplicating this information doesn't provide value.